run_tests.py

```diff
@@ -41,7 +41,7 @@
 import time
 
 
-COMPILE_CMD = ["bpy-addon-build", "--during-build", "dev"]
+COMPILE_CMD = ["bab", "-b", "dev"]
```

Comment:
Just curious, will a fast option be created now that we've upgraded to BpyBuild 0.4?

Reply:
Yeah that's my hope, I didn't get to implementing it fully though. I had it building fast by skipping the resource files, but then we still need to have at least the old copy of resource files retained in the addon folders (whereas right now, the builder always completely clears the initial install). So, some different workarounds like moving the pre-existing files to a temp place and then back into the b3d specific addons folder after is what we need to do. That will make testing so much faster, I've been missing it :)